market/iexchange.py

```diff
@@ -14,3 +14,18 @@ def name(cls) -> str:
             name of exchange
         """
         return ""
+
+    @classmethod
+    def fetchEntity(cls, entity, timeline) -> dict:
```

Comment:
`
```suggestion
    def fetchEntity(cls, entity: str, timeline: Timeline) -> Entity:
```
this method should return Entity object, otherwise Repository has to handle format of data per exchange. By putting this handling inside exchange, repository has abstract knowledge about the format of data coming from exchange.